compiler/test-resources/repl/parsing

```diff
@@ -0,0 +1,14 @@
+scala> ; 
+scala> ;; 
+scala> 1; 2 
+val res0: Int = 1
+val res1: Int = 2
+scala> 1; 
+val res2: Int = 1
+scala> 1;; 2 
+val res3: Int = 1
+val res4: Int = 2
+scala> } 
+1 | }
+  | ^
+  | eof expected, but '}' found
```

Comment:
@felixmulder Why do I need a space after `|`, even though when I run the repl on my machine it doesn't print the space. Is it a bug in the testing framework?

Reply:
Sounds like it!